admin_app/src/app/content/edit/page.tsx

```diff
@@ -214,14 +294,32 @@ const ContentBox = ({
     >
       <LanguageButtonBar
         expandable={true}
+        getLanguageList={() => {
+          return apiCalls.getLanguageList(token!);
+        }}
         onLanguageSelect={handleLanguageSelect}
         defaultLanguageId={content?.language_id || languageId}
         enabledLanguages={Object.keys(contentData).map(Number)}
         onMenuItemSelect={handleNewLanguageSelect}
         isEdit={true}
       />
       <Layout.Spacer multiplier={1} />
-      <Typography variant="body2">Title</Typography>
+      <Layout.FlexBox
+        {...appStyles.alignItemsCenter}
+        flexDirection={"row"}
+        justifyContent="space-between"
+      >
+        <Typography variant="body2">Title</Typography>
+        <IconButton
```

Comment:
Did that. But it still appears in edit mode when a language version is added is not added to the DB yet. I did that so there is no confusion in case someone picks a language by mistake and then click on another without saving the previous language version